packages/shorebird_cli/lib/src/release_type.dart

```diff
@@ -49,16 +49,22 @@ enum ReleaseType {
 
 extension ReleaseTypeArgs on ArgResults {
   Iterable<ReleaseType> get releaseTypes {
-    final List<String> releaseTypeCliNames;
+    List<String>? releaseTypeCliNames;
     if (wasParsed('platforms')) {
       releaseTypeCliNames = this['platforms'] as List<String>;
     } else {
-      final platformCliName = arguments.first;
-      if (ReleaseType.values
-          .none((target) => target.cliName == platformCliName)) {
-        throw ArgumentError('Invalid platform: $platformCliName');
+      if (arguments.isNotEmpty) {
+        final platformCliName = arguments.first;
+        if (ReleaseType.values
+            .none((target) => target.cliName == platformCliName)) {
+          throw ArgumentError('Invalid platform: $platformCliName');
+        }
+        releaseTypeCliNames = [platformCliName];
       }
-      releaseTypeCliNames = [platformCliName];
+    }
+
+    if (releaseTypeCliNames == null) {
+      return const [];
     }
```

Comment:
I don't think we need this early return. `map` on an empty list will return an empty list

Reply:
oh, but it is not checking on an empty list, it is checking on nullable map, we need an early return.

Either way, your comment made me think and I changed the early return for an elvis operator and a coalesce to empty list directly on the return and I guess it looks better!